### ISSUE TITLE
consider larger standard text wrap width

### ISSUE THREAD

**Issue by HolgerJeromin** [gravitystorm/openstreetmap-carto#1932]
http://www.openstreetmap.org/way/182816298#map=19/50.77169/6.03964
This amenity name has three lines on zoom 19. At zoom 18 this would be too low and is colliding with the kindergarden. This kindergarden has the `landcover-wrap-width` area size based.

**Comment by kocio-pl** [comment]
Now they don't collide, which is probably a side effect of upgrading Mapnik version on OSM servers. Could we close this issue now or we do want really to change text wrap width?

**Comment by pnorman** [comment]
:+1: to a wider wrap width

**Comment by kocio-pl** [comment]
@sommerluk What do you think about it? You're more into text-related things than me.

**Comment by sommerluk** [comment]
I agree with @HolgerJeromin and @pnorman that a bigger _text wrap width_ would improve the rendering here.

> is better for a map to have more square-like names than long wide lines.

No, not in general.

Typographically, for books you would say that between 8 and 14 words per line is comfortable to read. On maps, we have short labels (at least not whole paragraphs), so it might be better to have a somewhat smaller _line wrap_ than in books – maybe the half.

The main argument for “square-like” labels is that it works better when there is not enough space available on the map: Long labels create more collisions at the left and on the right (and less collisions on top and bottom), what makes easier the rendering of facilities that have a longer east-west extension, and makes more difficult the rendering of facilities that have a longer north-west extension. “Square-like” labels give us more balance in the collisions. So the reason for square-like labels is not that is would look good, but that it avoids bigger evil.

Currently, for example in the rendering of country labels or water body labels, we use rather square-like labels for small way_pixels areas (≙lower zoom levels) and a bigger _text wrap width_ for bigger way_pixels areas (≙once it’s high enough after zooming in so that we have probably enough space to render it with this bigger _line wrap width_ without collisions). At the same time we raise the font size and the line spacing, which makes visually clearer that all this space is occupied by this facility.

On the other hand, for shops (which are usually rather small features) we’ve opted for using the same line wrap on all zoom levels because even on highest zoom levels, there is often not enough space for a bigger _line wrap width_.

Further fine-tuning – like proposed for kindergarten here – seems useful to me (I just do not have so much time for that currently).
